src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/StoredProcedures/Replace/replaceWithoutVersionUpdate.js

```diff
@@ -0,0 +1,94 @@
+﻿/**
+* This stored procedure can be used to replace an existing document
+* 
+* @constructor
+* @param {any} doc - The CosmosResourceWrapper to save
+* @param {string} matchVersionId - required etag to match against when replacing an existing document
+*/
+function replaceWithoutVersionUpdate(doc, matchVersionId) {
```

Comment:
Do we need a stored proc if we're only updating a single document?

Reply:
Sorry this was supposed to be a draft PR. I am working on updating the stored procedure to update multiple docs at the same time.

Reply:
Removed the stored procedure and updated the code to use the SDK call for replace